### PR TITLE
📊 Add GBD dataset on Mental disorders prevalence rates

### PR DESCRIPTION
**Pull request by lucasrodes**
I am adding a subset of the IHME GBD 2019 dataset.

This is a temporary fix, as importing all IHME datasets is quite complex.

Having this dataset live is crucial to be able to update ~20 charts which are currently showing wrong values. For more context, read [this thread starting from the linked comment](https://github.com/owid/owid-issues/issues/760#issuecomment-1533730597).

### CODE REVIEW

File: etl/steps/data/garden/ihme_gbd/2023-05-15/gbd_mental_health_prevalence_rate.meta.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    variables:
+      prevalence_rate_anxiety_disorders:
+        title: Anxiety disorders (prevalence rate)
+        description: >-
+          Prevalence rate of anxiety disorders. This is estimated as the total number of cases with anxiety disorders per 100,000 population.
+        unit: "per 100,000 people"
+      prevalence_rate_bipolar_disorders:
+        title: Bipolar disorders (prevalence rate)
+        description: >-
+          Prevalence rate of bipolar disorders. This is estimated as the total number of cases with anxiety disorders per 100,000 population.
```
Comment (Marigold) [inline etl/steps/data/garden/ihme_gbd/2023-05-15/gbd_mental_health_prevalence_rate.meta.yml:14]: Shouldn't this be

> Prevalence rate of bipolar disorders. This is estimated as the total number of cases with ~anxiety~ **bipolar** disorders per 100,000 population.

(I guess you could also just leave it out and only keep "number of cases with disorders". It's clear from the first sentence)

**Review by Marigold — APPROVED** [review]
There's one small issue with metadata, otherwise looks good!

**Review by Marigold — APPROVED** [review]
There's one small issue with metadata, otherwise looks good!

**Comment by lucasrodes** [comment]
Thanks!

**Pull request merged**